integration_tests/test_cmath.py

```diff
@@ -0,0 +1,41 @@
+from cmath import (exp, log, log10, sqrt, acos, asin, atan, cos, sin, tan,
+                   acosh, asinh, atanh, cosh, sinh, tanh)
+from ltypes import c64
+
+def test_power_logarithmic():
+    x: c64
+    y: c64
+    x = complex(3, 3)
+    y = exp(x)
+    y = log(x)
+    y = sqrt(x)
+    y = log10(x)
```

Comment:
Let's also test the results in `y` to make sure we are getting the right answer and not 0. I tested locally and it seems to work. I think we need to implement this in LPython first:
```
>>> y = 3+3j
>>> y
(3+3j)
>>> y.real
3.0
>>> y.imag
3.0
```

And absolute value for complex numbers:
```
>>> abs(y - (3+3j))
0.0
>>> abs(y - (3+3.001j))
0.0009999999999998899
```

So let's do it in subsequent PRs.